deploy/helm/.helmignore

```diff
@@ -0,0 +1,15 @@
+.helmignore
+
+.git
+.gitignore
+.gitattributes
+
+.github/
+.vscode/
+
+opentelemetry-proto/*.md
+opentelemetry-proto/*.yaml
+opentelemetry-proto/Makefile
+opentelemetry-proto/LICENSE
+
```

Comment:
Personally, I'd exclude also protofiles, that we don't need. But I don't have a strong preference.

Reply:
Let's keep these files for now.